rogue-thi-app/data/calendar.json

```diff
@@ -9,220 +9,220 @@
     ]
   },
   {
-    "name": "Vorlesungsbeginn und Semesterstart",
-    "begin": "2022-03-15"
-  },
-  {
-    "name": "Vorlesungsfrei (Ostern)",
-    "begin": "2022-04-14",
-    "end": "2022-04-19"
-  },
-  {
-    "name": "Prüfungsanmeldung",
-    "begin": "2022-05-02",
-    "end": "2022-05-12"
-  },
-  {
-    "name": "Prüfungsabmeldung",
-    "begin": "2022-05-02",
-    "end": "2022-06-24"
-  },
-  {
-    "name": "Vorlesungsfrei (Christi Himmelfahrt, Brückentag)",
-    "begin": "2022-05-26",
-    "end": "2022-05-27"
-  },
-  {
-    "name": "Vorlesungsfrei (Pfingsten)",
-    "begin": "2022-06-03",
-    "end": "2022-06-07"
-  },
-  {
-    "name": "Vorlesungsfrei (Fronleichnam)",
-    "begin": "2022-06-16"
-  },
-  {
-    "name": "Rückmeldung",
-    "begin": "2022-07-02",
-    "end": "2022-08-07"
-  },
-  {
-    "name": "Erweiterter Prüfungszeitraum",
-    "begin": "2022-07-02",
-    "end": "2022-07-08"
-  },
-  {
-    "name": "Prüfungszeitraum",
-    "begin": "2022-07-09",
-    "end": "2022-07-19"
-  },
-  {
-    "name": "Notenbekanntgabe",
-    "begin": "2022-07-29T13:00",
-    "hasHours": true
+    "name": {
+      "de": "Semesterferien",
+      "en": "Semester break"
+    },
+    "begin": "2023-08-01",
+    "end": "2023-09-30"
   },
   {
-    "name": "Semesterferien",
-    "begin": "2022-07-30",
-    "end": "2022-09-30"
+    "name": {
+      "de": "Vorlesungsbeginn und Semesterstart",
+      "en": "Start of lectures and semester"
+    },
+    "begin": "2023-10-02"
   },
   {
-    "name": "Vorlesungsbeginn und Semesterstart",
-    "begin": "2022-10-04"
+    "name": {
+      "de": "Vorlesungsfrei",
+      "en": "Lecture-free"
```

Comment:
The THI just calls this "No lectures", I think we should stick to that [Source](https://www.thi.de/fileadmin/daten/studienangelegenheiten/Allgemeines/Semester_dates_studies_WS2324_und_SS24_-_16.03.23.pdf)